cell_classification/segmentation_data_prep.py

```diff
@@ -161,8 +170,8 @@ def prepare_example(self, data_folder, marker):
         # load and normalize the multiplexed image and masks
         mplex_img = self.get_image(data_folder, marker)
         mplex_img /= self.normalization_dict[marker]
-        binary_mask, instance_mask = self.get_instance_masks(
-            data_folder, self.cell_mask_key
+        binary_mask, instance_mask = self.get_instance_mask(
+            data_folder, self.segmentation_fname
```

Comment:
No need to pass the `self.segmentation_fname` to the function, can just access it within the function right?